src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -595,7 +595,10 @@ public sealed partial class Graphics : System.MarshalByRefObject, System.Drawing
         public static System.Drawing.Graphics FromHwndInternal(System.IntPtr hwnd) { throw null; }
         public static System.Drawing.Graphics FromImage(System.Drawing.Image image) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [Obsolete("Use one of the other overloads.")]
```

Comment:
Do we usually put breaking change docs for obsoletions, @ericstj ?

Reply:
Personally I think it wouldn't hurt.

Reply:
Yes, I've flagged the PR.  Should this use the new ObsoleteAttribute overloads with DiagnosticId?  Also the message seems rather light on detail.

Reply:
@ericstj I just used the text from the API review. Where should I follow up with DiagnosticId? It might be a bit overkill in this case as this is so specialized?

Reply:
For the text: Write some code that triggers the warning then show it to some folks and ask if they can understand what it means without googling or looking at reference source / docs.  Maybe include the method name?

For DiagnosticId: cc @jeffhandley or @terrajobst.  I found https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/docs/project/list-of-obsoletions.md
This also has some better warning text that should help and indicate the style used elsewhere.

> overkill in this case as this is so specialized

My understanding is that there is very little additional overhead to provide this: a single string parameter to the attribute,  and it enables a better user experience where folks don't just suppress all obsoletions.  Maybe others have a comment if we're permitting obsoletions without this id.

Reply:
Thanks, @ericstj. I'll dig in.